src/components/layout.js

```diff
@@ -14,33 +14,41 @@ import QuickThanks from "./quickthanks"
 import Reactions from './reactions'
 
 
-const Paywall = ({ floating }) => {
+const Paywall = ({ page }) => {
   const copyDiv = useRef(null)
 
-  useLayoutEffect(() => {
-    if (typeof window !== "undefined" && floating) {
-
-      if (
-        !window.localStorage.getItem("unlock_handbook") ||
-        !window.localStorage.getItem("sale_id")
-      ) {
-        const children = [].slice.call(document.getElementById('content').children);
-     
-        let isLocked = false;
-        for (const child of children) {
-          if (child.id === 'lock') isLocked = true;
-          if (isLocked === true ) {
-            child.style.display = 'none';
+
+  useEffect(() => {
+    if (
+      !window.localStorage.getItem("unlock_handbook") ||
+      !window.localStorage.getItem("sale_id")
+    ) {
+      setTimeout(
+        () => {
+          const children = [].slice.call(document.getElementById('content').children);
+       
+          let isLocked = false;
+          for (const child of children) {
+            if (child.id === 'lock') isLocked = true;
+            if (isLocked === true ) {
+              child.style.display = 'none';
+            }
           }
-        }
-      }
+
+        }, 2000
+      )
+    }
+  }, [page])
+
+  useLayoutEffect(() => {
+    if (typeof window !== "undefined" ) {
 
       window.requestAnimationFrame(() => {
-        const overlay = document.createElement("div")
-        const main = document.querySelector("main#content")
+        const overlay = typeof window !== 'undefined' && document.createElement("div")
+        const main    = typeof window !== 'undefined' && document.querySelector("main#content")
```

Comment:
You've already verified `window` exists by now so you don't have to check again.